src/renderer/components/+preferences/preferences.tsx

```diff
@@ -70,6 +82,21 @@ export class Preferences extends React.Component {
     }));
   }
 
+  minimapOptions: SelectOption<Boolean>[] = [
+    {
+      label: "Default",
+      value: null
+    },
```

Comment:
The default should be invisible to the user. Furthermore, if we remove this then we can use the toggle component instead of a dropdown.

Reply:
I'm not sure that it is really necessary. Default value hides minimap in the places, where we don't expect more data than one page (kube-config/ details).